src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsConfig.java

```diff
@@ -105,6 +122,22 @@ public void addGlobalVarPasswordPair(VarPasswordPair varPasswordPair) {
     }
 
     /**
+     * Adds a regex at the global level.
+     *
+     * <p>If regex is blank (as defined per the Commons Lang
+     * library), then the pair is not added.</p>
+     *
+     * @since 2.9
+     */
+    public void addGlobalVarMaskRegex(VarMaskRegex varMaskRegex) {
+        // blank values are forbidden
+        if(StringUtils.isBlank(varMaskRegex.getRegex())) {
```

Comment:
The original code in addGlobalVarPasswordPair() is not perfect. Maybe makes sense to log the message at FINE level at least